turborepo-tests/integration/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "turborepo-tests-integration-common",
+  "name": "turborepo-tests-integration",
```

Comment:
I think I'm missing something because I don't see this package name referenced anywhere, so I'm not sure how it was running these tests